Google/google_module/google_drive_reports.py

```diff
@@ -88,10 +88,11 @@ def run(self):
                 try:
                     self.get_reports_events()
 
-                except HTTPError | BaseHTTPError as ex:
-                    self.log_exception(ex, message="Failed to get next batch of events")
```

Comment:
Could you keep this block?

Reply:
I'm wondering if it's this line who causes the error. ok I will keep it.